pkg/services/ngalert/notifier/alertmanager.go

```diff
@@ -552,5 +589,283 @@ func (am *Alertmanager) GetAvailableNotifiers() []*alerting.NotifierPlugin {
 				},
 			},
 		},
+		{
+			Type:        "pagerduty",
+			Name:        "PagerDuty",
+			Description: "Sends notifications to PagerDuty",
+			Heading:     "PagerDuty settings",
+			Options: []alerting.NotifierOption{
+				{
+					Label:        "Integration Key",
+					Element:      alerting.ElementTypeInput,
+					InputType:    alerting.InputTypeText,
+					Placeholder:  "Pagerduty Integration Key",
+					PropertyName: "integrationKey",
+					Required:     true,
+					Secure:       true,
+				},
+				{
+					Label:   "Severity",
+					Element: alerting.ElementTypeSelect,
+					SelectOptions: []alerting.SelectOption{
+						{
+							Value: "critical",
+							Label: "Critical",
+						},
+						{
+							Value: "error",
+							Label: "Error",
+						},
+						{
+							Value: "warning",
+							Label: "Warning",
+						},
+						{
+							Value: "info",
+							Label: "Info",
+						},
+					},
+					PropertyName: "severity",
+				},
+				{ // New in 8.0.
+					Label:        "Class",
+					Description:  "The class/type of the event, for example 'ping failure' or 'cpu load'",
+					Element:      alerting.ElementTypeInput,
+					InputType:    alerting.InputTypeText,
+					PropertyName: "class",
+				},
+				{ // New in 8.0.
+					Label:        "Component",
+					Description:  "Component of the source machine that is responsible for the event, for example mysql or eth0",
+					Element:      alerting.ElementTypeInput,
+					InputType:    alerting.InputTypeText,
+					Placeholder:  "Grafana",
+					PropertyName: "component",
+				},
+				{ // New in 8.0.
+					Label:        "Group",
+					Description:  "Logical grouping of components of a service, for example 'app-stack'",
```

Comment:
I think prometheus AM makes such properties templateable as too, is it the case here too?

Reply:
Yes, it can be template here.